component/src/main/java/org/cbioportal/genome_nexus/component/annotation/VariantClassificationResolver.java

```diff
@@ -46,6 +47,14 @@ public String resolve(VariantAnnotation variantAnnotation, TranscriptConsequence
                 this.consequencePrioritizer.pickHighestPriorityConsequence(transcriptConsequence.getConsequenceTerms()),
                     variantType, isInframe);
         }
+        // if variant has no transcript and has intergenicConsequences instead, try to resolve the variant classification from intergenicConsequences
+        else if (variantAnnotation != null && variantAnnotation.getIntergenicConsequences() != null && !variantAnnotation.getIntergenicConsequences().isEmpty()) {
+            // pick the highest priority consequence from the intergenic consequences
+            // get every getConsequenceTerms() from each intergenic consequence and put in a list
+            variantClassification = this.resolveVariantClassification(
+                this.consequencePrioritizer.pickHighestPriorityConsequence(variantAnnotation.getIntergenicConsequences().stream().flatMap(consequence -> consequence.getConsequenceTerms().stream()).collect(Collectors.toList())),
+                    variantType, isInframe);
```

Comment:
It would be easier to read this way, but what we actually need is a prettier equivalent for Java.

```suggestion
            variantClassification = this.resolveVariantClassification(
                this.consequencePrioritizer.pickHighestPriorityConsequence(
                    variantAnnotation
                    .getIntergenicConsequences()
                    .stream()
                    .flatMap(consequence -> consequence.getConsequenceTerms().stream())
                    .collect(Collectors.toList())
                ),
                variantType,
                isInframe
            );
```